openretina/cli/train.py

```diff
@@ -10,16 +10,29 @@
 
 from openretina.data_io.base import compute_data_info
 from openretina.data_io.cyclers import LongCycler, ShortCycler
+import platform
+import sys
+import mlflow
 
 log = logging.getLogger(__name__)
 
 
-@hydra.main(version_base="1.3", config_path="../../configs", config_name="hoefling_2024_core_readout_high_res")
-def main(cfg: DictConfig) -> None:
-    train_model(cfg)
+@hydra.main(
+    version_base="1.3",
+    config_path="../../configs",
+    config_name="hoefling_2024_core_readout_high_res",
+)
+def main(cfg: DictConfig) -> float | None:
+    score = train_model(cfg)
+    if cfg.objective_target is not None:
+        return score
+    return None
 
 
-def train_model(cfg: DictConfig) -> None:
+def train_model(cfg: DictConfig) -> float | None:  ## normally train_model
+    # Get the current trial number
+    # Update the trial number in the configuration
```

Comment:
maybe delete the comments or move them to the location where that happens.
If you want to add a longer description what the function does it's best to use docstrings like:
```suggestion
def train_model(cfg: DictConfig) -> float | None:
   """ Trains a model and returns a score in case the xyz config parameter was set or so """
```

Reply:
This is just an old comment I forgot to delete, now deleted